### ISSUE TITLE
python manage.py migrate

### ISSUE THREAD

**Issue by aaronweaver** [rackerlabs/django-DefectDojo#56]
Ran python manage.py migrate and have this error:
 
<snip>
Applying auditlog.0006_object_pk_index...Traceback (most recent call last):
.
.
.
django.db.utils.OperationalError: (1170, "BLOB/TEXT column 'object_pk' used in key specification without a key length")
</snip>

**Comment by rackGregA** [comment]
@grendel513 I think I remember us seeing this issue somewhere else as well, do you remember or recall the solution? If not I'll dig in this afternoon.

**Comment by devGregA** [comment]
@grendel513. Aaron emailed me this morning and also told me it was a fresh install. Will take a look when I get back from lunch

**Comment by devGregA** [comment]
Hi @aaronweaver what operating system are you using? I'm free to try and reproduce the error now.

**Comment by devGregA** [comment]
@aaronweaver, just to confirm it was a new install?
If it was you don't need to use migrate. You only need to migrate when a new release requires database changes.

bash setup.bash
python manage.py bower install
python manage.py collectstatic
python manage.py runserver

Should be all you need for new instances.

**Comment by aaronweaver** [comment]
@devGregA Let me try again and yes this is a fresh install.

**Comment by aaronweaver** [comment]
@devGregA I started fresh and installed correctly. Thanks for your assistance!

**Comment by devGregA** [comment]
No prob! Anytime @aaronweaver, happy to help.
